src/braket/analog_hamiltonian_simulator/rydberg/numpy_solver.py

```diff
@@ -38,7 +38,7 @@ def rk_run(
 
     Args:
         program (Program): An analog simulation program for a Rydberg system
-        configurations (List[str]): The list of configurations that comply with the
```

Comment:
For the public functions that you changed, to keep backward compatibility, can/should we support both a List and a Dict? 

Also, could you please update the docs the explain the possible values (especially adding what the Dict is supposed to look like, perhaps with an example).